Resources/config/exception_listener.xml

```diff
@@ -5,13 +5,22 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="fos_rest.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
-            <tag name="kernel.event_subscriber" />
+        <service id="security.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
```

Comment:
OK - could you perhaps give me the service ID or class which is the one you'd like decorated? (Possibly a quick technique for finding that out if you'd like me to?)

or is this request more about just the configuration in the link you've provided?